app/views/catalog/_index_default.html.erb

```diff
@@ -0,0 +1,18 @@
+<%# OVERRIDE Hyrax 3.4.2: change layout for metadata %>
+
+<% doc_presenter = index_presenter(document) %>
+<%# default partial to display solr document fields in catalog index view -%>
+<dl class="document-metadata dl-horizontal dl-invert">
```

Comment:
I'm a bit surprised to see a `dl` tag without nested `dt` or `dd` elements.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

My read is that a `dl` tag should have `dt` and `dd` elements and not `p` elements as direct ancestors.

This is a non-blocker as HTML is forgiving, but one to consider as the `dl` announces differently to screen readers.

Reply:
@jeremyf - Thank you for this comment. I copied [this partial in from Blacklight](https://github.com/projectblacklight/blacklight/blob/v6.24.0/app/views/catalog/_index_default.html.erb) and wanted to make as few changes as possible. I changed the interior of the `<dl>` which got rid of the `<dt>` and `<dd>`. I will update to use the div instead and make any necessary changes. Thank you!